src/components/Main/Team/AddYourself.js

```diff
@@ -11,19 +11,30 @@ export const AddYourself = (props) => {
     ...otherProps
   } = props
 
+  const handleInputChange = (event) => {
+    const file = event.target.files[0];
+
+    if (file) {
+        const reader = new FileReader();
+        reader.onload = function (e) {
+            image.style.display = 'block'
+            image.src = e.target.result;
+            image.style.display = 'block';
+        };
+        reader.readAsDataURL(file);
+    }
+  
+  };
 
   const renderListItem = () => {
     return (
       <>
         <div className={'team__person you'}>
-          <div className={'team__photo'}><input type="file" id="fileInput" accept="image/*"></input>
-            <img id="image" /></div>
+          <div className={'team__photo'}>
+            <img id="image" className={'you'}/></div>
+            <input type="file" id="fileInput" accept="image/*" onChange={handleInputChange} ></input>
           <div className={'team__function'}><input type="text" id="textInput" placeholder={'Please write your function'}></input></div>
           <div className={'team__data'}><input type="text" id="textInput" placeholder={'Please write your name'}></input></div>
```

Comment:
Pomysł fajny, ale nie do końca wiem jak ma to działać - wypełniłem dane, ale nic się nie stało :p

![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/b9d0220f-8fc4-4f9b-8841-94fe84b2a89d)